ide/build.gradle

```diff
@@ -30,6 +38,7 @@ oomphIde {
 	addProjectFolder('../org.eclipse.lsp4j')
 	addProjectFolder('../org.eclipse.lsp4j.generator')
 	addProjectFolder('../org.eclipse.lsp4j.jsonrpc')
+	addProjectFolder('../org.eclipse.dsp4j.jsonrpc')
```

Comment:
I'm not sure about the namespace `org.eclipse.dsp4j`, since theoretically that could cause name clashes with other Eclipse projects. On the other hand, I like the symmetry with `org.eclipse.lsp4j`. What do others think about this?

An alternative project name could be `org.eclipse.lsp4j.jsonrpc.debug`

Reply:
OK, depending on decision here will affect name of the interfaces containing plug-in. I was originally planning on org.eclipse.dsp4j, but perhaps org.eclipse.lsp4j.debug makes more sense.

Reply:
Yes, IIRC there even is some rule at eclipse for java packages to use the project name as prefix.